### PR TITLE
cip67 from lib

### PR DESCRIPTION
**Pull request by zuzunker**
replaced local implementation of resolving ascii asset name and resolving cip67 tags with lib calls

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscript] ⚠️ `ruslan/cip67-yoroi-lib` doesn't full our branch naming convention.
Should follow `<tag>/<jira-ticket-id>/<short-name>`
<details>
<summary>Details</summary>

- `<tag>`: `fix`, `feat`, `chore`, `refactor`, `test`, ...
- `<jira-ticket-id>`: The issue ID of JIRA all capital letters
- `<short-name>`: A short and descriptive name for the branch
</details>

[](__JIRA_COMMENT_FLAG)

**Review by zuzunker — COMMENTED** [review]
/check

**Review by yushih — DISMISSED** [review]

File: packages/yoroi-extension/app/stores/stateless/tokenHelpers.js [hunk 1]
```diff
@@ -26,62 +25,21 @@ export function getTokenName(
...
-          hexName: bytes.subarray(4).toString('hex'),
-        };
-      }
-    }
+function resolveNameProperties(name: ?string): { name: string, cip67Tag: ?string } {
+  if (name == null || name.length === 0 || !isHexadecimal(name)) {
+    return { name: '', cip67Tag: null };
   }
+  const { asciiName, hexName, cip67Tag } =
+    AssetNameUtils.resolveProperties(name ?? '');
```
Comment (yushih) [inline packages/yoroi-extension/app/stores/stateless/tokenHelpers.js]: `?? ''` seems not necessary because previous `if` covers this case

**Review by Nebyt — DISMISSED** [review]

File: packages/yoroi-extension/app/stores/stateless/tokenHelpers.js (hunk 1 above)
Comment (zuzunker) [reply packages/yoroi-extension/app/stores/stateless/tokenHelpers.js]: Ah right! A remnant from before the check got added

**Pull request merged**